akka-actor/src/main/scala/akka/actor/ActorPath.scala

```diff
@@ -8,6 +8,47 @@ import akka.japi.Util.immutableSeq
 import java.net.MalformedURLException
 import java.lang.{ StringBuilder ⇒ JStringBuilder }
 
+/**
+ * Java API
+ */
+object ActorPaths {
```

Comment:
AFAIR it needs to be a "companion", that is have a `class` of the same name for the static forwarders to be generated?